README.md

```diff
@@ -1 +1,106 @@
-# Nextcloud-OpenProject-App repository
\ No newline at end of file
+# OpenProject as Nextcloud's External App
+
+## Manual Installation
+For the manual installation of `OpenProject` as an external application of `Nextcloud`, make sure that your `Nextcloud` as well as `OpenProject` instance is up and running.
+
+### 1. Install `app_api` application
+
+Assuming you’re in the apps folder directory:
+
+- Clone
+	```bash
+	git clone https://github.com/cloud-py-api/app_api.git
+	```
+- build
+   ```bash
+   cd app_api
+   npm ci && npm run dev
+    ```
+- Enable the `app_api`
+	```bash
+	# Assuming you’re in nextcloud server root directory
+	sudo -u www-data php occ a:e app_api
+	```
+ 
+### 2. Register deploy daemons (In Nextcloud)
+
+- Navigate to `Administration Settings > AppAPI`
+- Click register daemons
+- Select `Manual Install` for Daemon configuration template
+- Put `manual_install` for name and display name
+- Deployment method as `manual-install`
+- Daemon host as `localhost`
+- Click Register
```

Comment:
Nextcloud URL is auto generated but should we need to mention about that or not

Reply:
if it is autogenerated then no need to